integration-tests/gradle/src/main/resources/grpc-multi-module-project/build.gradle

```diff
@@ -1,5 +1,7 @@
 buildscript {
     repositories {
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
What was the reason for this move?

Reply:
Right, this one can be reverted, it does not make sense